.github/workflows/build.yml

```diff
@@ -115,6 +115,11 @@ jobs:
     name: End-to-end publish packages
     runs-on: ${{matrix.os}}
     steps:
+      - name: Setup runner
+        if: ${{matrix.os == 'windows-latest'}}
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
```

Comment:
This fix the 1st issue: Inconsistent end of line between the published version and the verdaccio version